src/cordova/apple/xcode/ios/Outline/Outline.entitlements

```diff
@@ -11,7 +11,6 @@
 	<key>com.apple.security.application-groups</key>
 	<array>
 		<string>group.org.outline.ios.client</string>
-		<string>$(TeamIdentifierPrefix)org.outline.macos.client</string>
```

Comment:
@sbruens will this break mac Catalyst?

I don't understand why having this would break the build, since an app can be in multiple application groups. Perhaps we need to configure the application group to include the ios app.

Reply:
This will most likely break macOS, because they share this file.
Please wait for @sbruens to chime in.
Other than that, Makefile and pbxproj look good.

Reply:
It will break catalyst because this app group is used in several places to share data between the network extension and the host app, like the logs directory for Sentry and the tunnel store (see https://github.com/search?q=repo%3AJigsaw-Code%2Foutline-client%20%22QT8Z3Q9V3A.org.outline.macos.client%22&type=code).

Those values need to match the app groups the app has access to. I think we may be able to move catalyst over to use the ios app group but it would mean that the catalyst app would no longer use the existing mac paths for logs and tunnel store preferences, but that might be ok?

Reply:
Btw I added those entitlements very recently: https://github.com/Jigsaw-Code/outline-client/pull/1811

Reply:
Perhaps the iOS app provisioning profile needs to be updated online to add those groups?

Reply:
I think the problem is that macOS groups are special and not managed in the Developer portal (https://developer.apple.com/documentation/xcode/configuring-app-groups#Create-App-Groups-for-macOS-apps). So you can't even create an app group that doesn't start with `group.`.

That app groups shouldn't be set on the iOS app anyway. But the XCode UI is weird where it automatically adds macOS app groups to the iOS section (but not the other way around?). Is there a way to set them in the release script?

I guess the best course of action would be to update those hard-coded values and move catalyst over to the iOS app group.

Reply:
I don't think so. They are not stored under the app group.

@daniellacosse you should be able to check that by running the catalyst app after you change these values and checking that your servers show up.